proto-lens/src/Data/ProtoLens/Encoding/Growing.hs

```diff
@@ -0,0 +1,59 @@
+-- | A mutable vector that grows in size.
```

Comment:
Does it make sense to give a quick code sample of how to use this module safely?

Reply:
Done.